Python/Product/Analysis/Values/FunctionInfo.cs

```diff
@@ -499,23 +499,25 @@ private class StringArrayComparer : IEqualityComparer<string[]> {
             varRef.AddTypes(unit, value);
         }
 
-        public override IAnalysisSet GetMember(Node node, AnalysisUnit unit, string name) {
-            // Must unconditionally call the base implementation of GetMember
-            var ignored = base.GetMember(node, unit, name);
```

Comment:
The base class implementation doesn't do anything anymore, so at some point these comments just became stale.